.github/workflows/unittest.yml

```
@@ -0,0 +1,26 @@
+name: unit-test
+
+on: [push, pull_request]
```

Today I discovered a better setting for this is
```suggestion
on:
  workflow_dispatch:
  pull_request:
  push:
    branches: [next, qa, main]
```